doc/source/ray-core/scheduling/accelerators.rst

```diff
@@ -26,7 +26,7 @@ The accelerators natively supported by Ray Core are:
      - Experimental, supported by the community
    * - `AWS Neuron Core <https://awsdocs-neuron.readthedocs-hosted.com/en/latest/general/arch/model-architecture-fit.html>`_
      - neuron_cores
-     - Experimental, supported by the community
+     - Experimental, supported by the community. See examples on AWS EKS `here <https://awslabs.github.io/data-on-eks/docs/gen-ai>`_
```

Comment:
```suggestion
     - Experimental, supported by the community. See `Generative AI on EKS <https://awslabs.github.io/data-on-eks/docs/gen-ai>`_ for examples.
```

Reply:
Thanks for adding the link. We try to avoid using "here" for links because it's not very descriptive. Would you mind installing Vale, which will catch these style nuances? go/vale